tests/TestCase/Core/GlobalFunctionsTest.php

```diff
@@ -0,0 +1,371 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         3.0.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Core;
+
+use Cake\Core\Configure;
+use Cake\Http\Response;
+use Cake\TestSuite\TestCase;
+use stdClass;
+
+require_once CAKE . 'Core/functions_global.php';
```

Comment:
Including this file means if one of these gets functions gets used without namespace in files which are loaded after this test case, we won't we able to detect ir.

Perhaps using the `@runTestsInSeparateProcesses` for the test class can avoid that?